### PR TITLE
Stripe Webhooks

### PR DESCRIPTION
**Pull request by mariofix**
* Webhooks support for stripe.StripeProviderV3
* internal file order

### CODE REVIEW

File: docs/backends.rst [hunk 1]
```diff
@@ -213,22 +213,49 @@ Example::
...
+To use this you need to set the option PAYMENT_HOST to match the public URL for your Django Project
+
+Then you need to go to your Stripe Dashboard, then activate Test Mode, finally go to Developers -> Webhooks.
+Click on the button "+ Add endpoint", set the URL from PAYMENT_HOST and add "payments/process/stripe", so your
+final URL should look like this https://example.com/payments/process/stripe
+You must select the following events:
+* checkout.session.async_payment_failed
+* checkout.session.async_payment_succeeded
+* checkout.session.completed
+* checkout.session.expired
```
[the remark this answers is not in the document]
Comment (mariofix) [reply docs/backends.rst]: No, Stripe only works with the URL preconfigured. PayPal supports both, it depends on the broker. That's why it's Stripe specific instructions, I was trying to get to the mercado pago dashboard to add the config for it.

File: docs/backends.rst [hunk 2]
```diff
@@ -250,3 +281,119 @@ Example::
...
+        # Implement payment form rendering logic
+        pass
+
+    def capture(self, payment, amount=None):
+        # Implement payment capture logic
+        raise NotImplementedError("Capture method not implemented.")
+
+    def refund(self, payment, amount=None):
+        # Implement payment refund logic
+        raise NotImplementedError("Refund method not implemented.")
```
Comment (WhyNotHugo) [inline docs/backends.rst:357]: Missing one level of indentation here.

**Comment by mariofix** [comment]
> CI failures seem unrelated

yeah, I though they were because of the dj22 drop, need help with these errors?

**Review by WhyNotHugo — APPROVED** [review]

**Comment by WhyNotHugo** [comment]
Thanks!

**Pull request merged**